mappings/net/minecraft/client/option/Option.mapping

```diff
@@ -44,33 +47,37 @@ CLASS net/minecraft/unmapped/C_cyyavvmb net/minecraft/client/option/Option
 	FIELD f_lcqyoyqc SIMULATION_DISTANCE Lnet/minecraft/unmapped/C_lszwfdec;
 	FIELD f_lkhqlgwg ENTITY_SHADOWS Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_lqlrrfpe ALLOW_SERVER_LISTINGS_TOOLTIP Lnet/minecraft/unmapped/C_rdaqiwdt;
-	FIELD f_mmyysnbc SPRINT_TOGGLED Lnet/minecraft/unmapped/C_hxxnafqd;
-	FIELD f_msjljtln TOUCHSCREEN Lnet/minecraft/unmapped/C_hxxnafqd;
+	FIELD f_mmyysnbc TOGGLEABLE_SPRINT Lnet/minecraft/unmapped/C_hxxnafqd;
+	FIELD f_msjljtln TOUCHSCREEN_MODE Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_msxpgfqp REDUCED_DEBUG_INFO Lnet/minecraft/unmapped/C_hxxnafqd;
-	FIELD f_npempxwa AO Lnet/minecraft/unmapped/C_hxxnafqd;
+	FIELD f_npempxwa SMOOTH_LIGHTNING Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_pcvepgzp CHAT_WIDTH Lnet/minecraft/unmapped/C_lszwfdec;
 	FIELD f_pyvkmaaz TOGGLE_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_qeexjcmi VIEW_BOBBING Lnet/minecraft/unmapped/C_hxxnafqd;
-	FIELD f_qhphpafk SENSITIVITY Lnet/minecraft/unmapped/C_lszwfdec;
+	FIELD f_qhphpafk MOUSE_SENSITIVITY Lnet/minecraft/unmapped/C_lszwfdec;
+	FIELD f_qhxpolie PRIORITIZE_CHUNK_UPDATES_NEARBY_TOOLTIP Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_qmzvovyl CHAT_OPACITY Lnet/minecraft/unmapped/C_lszwfdec;
 	FIELD f_qnibjufu AUDIO_DEVICE Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_rmcclttf TEXT_BACKGROUND_OPACITY Lnet/minecraft/unmapped/C_lszwfdec;
 	FIELD f_snqiyhzz AUTO_JUMP Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_tizxpbgc GUI_SCALE Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_uufqfupo ATTACK_INDICATOR Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_uytcbrmi CHAT_LINE_SPACING Lnet/minecraft/unmapped/C_lszwfdec;
-	FIELD f_vapirxpd SNEAK_TOGGLED Lnet/minecraft/unmapped/C_hxxnafqd;
+	FIELD f_vapirxpd TOGGLEABLE_SNEAK Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_vggvgfee FULLSCREEN Lnet/minecraft/unmapped/C_hxxnafqd;
+	FIELD f_wjznjfkb PRIORITIZE_CHUNK_UPDATES_BY_PLAYER_TOOLTIP Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_wxmgecbh RAW_MOUSE_INPUT Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_wygotzue MONOCHROME_LOGO_TOOLTIP Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_xjcbmfsh INVERT_MOUSE Lnet/minecraft/unmapped/C_hxxnafqd;
-	FIELD f_yerpiqmb CHUNK_UPDATES_PRIORITIZATION Lnet/minecraft/unmapped/C_hxxnafqd;
+	FIELD f_yerpiqmb PRIORITIZE_CHUNK_UPDATES Lnet/minecraft/unmapped/C_hxxnafqd;
 	FIELD f_zgszdkos key Lnet/minecraft/unmapped/C_rdaqiwdt;
 	METHOD <init> (Ljava/lang/String;)V
 		ARG 1 key
+	METHOD m_adztwlyp (Lnet/minecraft/unmapped/C_lscptbhn;)Lnet/minecraft/unmapped/C_inrattee;
+		ARG 0 gameOptions
 	METHOD m_ahgwvofk (Lnet/minecraft/unmapped/C_lscptbhn;Ljava/lang/Double;)V
 		ARG 0 gameOptions
-		ARG 1 chatOpacity
+		ARG 1 value
```

Comment:
Not sure how I feel about `someName` -> `value`, but it is nice that it is less likely to be mismatched. The context around should be fine, but it is sometimes nice to not have to look at the context. This is probably fine, but some other input would be nice.

Reply:
Considering that those are parameters of a lambda function, I don't think it'll really matter; But yeah, if people think that having that format is better, then I'll flip the direction of the standardization

Reply:
I think it's fine as `value`